### PR TITLE
Multimeter logging / playback added

### LINKED ISSUE #1863: Multimeter log data is not being added to realm hence to shown in Data logger Activity
**Actual Behaviour**
Multimeter log data is not being added to realm hence to shown in Data logger Activity. So user can not play back the log

**Expected Behaviour**
Saved log should be added to realm and shown in data logger activity , so user can play back it.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1863 
**Changes**: adding multimeter log data to realm

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:
[multimeter_log.zip](https://github.com/fossasia/pslab-android/files/3389738/multimeter_log.zip)

### CODE REVIEW

File: app/src/main/java/io/pslab/interfaces/sensorloggers/MultimeterRecordables.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package io.pslab.interfaces.sensorloggers;
+
+import io.pslab.models.MultimeterData;
+import io.realm.RealmResults;
+
+public interface MultimeterRecordables {
+    MultimeterData getMultimeterData(long timeStamp);
+
```
Comment (cweitat) [inline app/src/main/java/io/pslab/interfaces/sensorloggers/MultimeterRecordables.java]: please remove excessive spaces

**Comment by neel1998** [comment]
@CloudyPadmal @cweitat please review and merge

**Comment by neel1998** [comment]
@CloudyPadmal please review so this can be merged

File: app/src/main/java/io/pslab/models/PSLabSensor.java [hunk 2]
```diff
@@ -109,12 +83,31 @@
     public static final String ROBOTIC_ARM = "Robotic Arm";
     public static final String WAVE_GENERATOR = "Wave Generator";
     public static final String OSCILLOSCOPE = "Oscilloscope";
-
+    public static final String MULTIMETER = "Multimeter";
+    public final String KEY_LOG = "has_log";
+    public final String DATA_BLOCK = "data_block";
+    public boolean isRecording = false;
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/models/PSLabSensor.java]: Why all these changed? Previous state has them in a neat way. Revert

File: app/src/main/java/io/pslab/activity/MultimeterActivity.java [hunk 3]
```diff
@@ -110,8 +116,17 @@
     private boolean locationEnabled = true;
     private long recordPeriod;
     private double lat = 0, lon = 0;
-    private String multimeterCSVheader = "Data,Value,Latitude,Longitude\n";
+    private String multimeterCSVheader = "Timestamp,DateTime,Data,Value,Latitude,Longitude";
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/activity/MultimeterActivity.java]: Why are we adding a timestamp and a datetime?

Comment (neel1998) [reply app/src/main/java/io/pslab/activity/MultimeterActivity.java]: To have consistent xsv format as other instruments

File: app/src/main/java/io/pslab/models/PSLabSensor.java (hunk 2 above)
Comment (neel1998) [reply app/src/main/java/io/pslab/models/PSLabSensor.java]: Done :+1:

**Pull request merged**